changelog.d/9802.doc

```diff
@@ -0,0 +1 @@
+Hardened systemd files as proposed in Issue 9760 and added them to contrib. Changed the docs to reflect the presence of these files in contrib.
```

Comment:
```suggestion
Add hardened systemd files as proposed in [#9760](https://github.com/matrix-org/synapse/issues/9760) and added them to `contrib/`. Change the docs to reflect the presence of these files.
```

Just some minor changes. Also some contributors put a "Contributed by @abc." at the end, which you can do if you want :slightly_smiling_face: